tests/integration/fixtures/ok-source-extract/module.grepout

```diff
@@ -0,0 +1,6 @@
+<p>You can put <em>markdown</em> in triple-quoted strings in Python.</p>
+<p>It works in docstrings, too. The start and end quotes must be on their own lines.
+Drawback: <code>simple</code> does not remove leading whitespace.</p></div>
```

Comment:
Shouldn't the "replace" option be able to remove leading whitespace?

I imagine this would be an issue in both C++ and python comments.

Reply:
Righto. The issue here is knowing how _much_ whitespace for "replace" to strip. Typically thatwould involve knowing how much whitespace there was at the beginning of the line of the "start" (and/or "resume", if we use it) marker. So I could save the start/resume match and allow backreferences to what it matched in the "replace" facility, but I didn't want to get too fancy on the first pass and I haven't actually used that facility yet. But I would be fine putting it in if you want me to -- let me know.

Reply:
I wasn't thinking anything so fancy.  Markdown has limited use for indentation, so one could conceivably remove all whitespace from the beginning of the lines.  I'm ok with not including it and leave it to the user.

Reply:
Fine; you can definitely remove all leading whitespace with the existing 'replace' facility, so I will leave this aspect be, at least for now.